src/deflate/mod.rs

```diff
@@ -4,10 +4,7 @@
 //!
 //! # Examples
 //! ```
-//! #[cfg(feature = "no_std")]
-//! use core2::io::{Read, Write};
-//! #[cfg(not(feature = "no_std"))]
-//! use std::io::{Read, Write};
+//! use core2:io::{Read, Write};
```

Comment:
```suggestion
//! use core2::io::{Read, Write};
```
Minor typo, similar typo repeated in other places.

Reply:
Yes, no need to repeat.